pkg/com.github.hydr8gon.NooDS.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.hydr8gon.NooDS</id>
+  <launchable type="desktop-id">com.github.hydr8gon.NooDS.desktop</launchable>
+  <name>NooDS</name>
+  <summary>A (hopefully!) speedy NDS emulator</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <developer_name>Hydr8gon</developer_name>
+  <content_rating type="oars-1.0" />
+  <description>
+    <p>
+      The goal of NooDS is to be a fast and portable Nintendo DS emulator. It's not quite there speed-wise, but it does offer most other features that you might expect from a DS emulator. It even supports GBA backwards compatability! I'm doing this for fun and as a learning experience, and also because I'm a huge fan of the DS. It may not be a worthy competitor for the other DS emulators just yet, but I believe that I can get it there someday. If not, that's fine too; like I said, I'm just having fun!
```

Comment:
This is pretty long; I'd maybe cut it off after the GBA backwards compatibility part. Although, what is this file for in the first place? It seems like a good portion of my applications don't have this at all. The minimalist in me kinda wants to omit this file entirely if it's not that important. Then we can also plop `noods.desktop` right in the root directory, like similar files for other platforms are.

Reply:
This file is documented [here](https://docs.appimage.org/packaging-guide/optional/appstream.html). One notable place that I've seen it used is appimage.github.io. [Here](https://appimage.github.io/Armagetron_Advanced/) is what a package that provides an appdata.xml file looks like, and [here](https://appimage.github.io/clownmdemu/) is one that does not. The latter is considerably more barebones due to lacking metadata the extra metadata that this type of file provides.